account_bank_statement_import_auto_reconcile/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Automatic reconciliation after import",
+    "version": "10.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": 'Banking addons',
+    "summary": "This module allows you to define automatic "
+    "reconciliation rules to be run after a bank statement is imported",
+    "depends": [
+        'account_bank_statement_import',
+        'base_domain_operator',
```

Comment:
Is there a PR to migrate this?

Reply:
Yes, its https://github.com/OCA/server-tools/pull/1525 but first this fix https://github.com/hbrunn/server-tools/pull/5 has to be merged.